src/mame/drivers/lwriter.cpp

```diff
@@ -436,6 +490,16 @@ uint8_t lwriter_state::via_pb_lw2nt_r()
 
 void lwriter_state::via_pb_w(uint8_t data)
 {
+	if ((data & 0x20) && ((m_via_pb & 0x20) == 0)) {
+		m_reset_count++;
+		logerror("reset fifo %d pb6 %d\n", m_fifo_count, m_pb6_tick_count);
```

Comment:
Unconditional logerror() for things that happen every frame or more often hurts emulation performance, even when the logging isn't being sent anywhere.  You should look into the LOGMASKED() framework in logmacro.h so that it's not logging when you aren't debugging it, and you can choose what gets logged.  A recent example by me is in src/devices/machine/spi_sdcard.cpp.

Reply:
Done